src/pages/CoinListPage/components/CoinList/CoinList.tsx

```diff
@@ -23,64 +25,54 @@ const CoinList = ({
   paginationHide = false,
   /** Количество контента */
   contentCount,
+  /**  */
+  paginationStore,
 }: CoinListProps) => {
-  const {
-    firstContentIndex,
-    lastContentIndex,
-    nextPage,
-    prevPage,
-    page,
-    setPage,
-    totalPages,
-  } = usePagination({
-    contentPerPage: 10,
-    count: contentCount,
-  });
-
   const labels: string[] = [];
   for (let i = 0; i < contentCount; i++) {
     labels[i] = `${i}`;
   }
```

Comment:
чтобы не пересчитывать в компоненте на каждый ререндер это значение стоит использовать [хук useMemo](https://reactjs.org/docs/hooks-reference.html#usememo), а еще было бы удобно заюзать метод массива `.map()`
```suggestion
  const labels = React.useMemo(
    () => Array.from({ length: contentCount }, (_, index) => String(index)),
    [contentCount]
  );
```